test/evaluation/limitDisclosureEvaluationHandler.spec.ts

```diff
@@ -16,20 +15,20 @@ describe('evaluate', () => {
 
   it('should return ok if verifiablePresentation doesn\'t have the etc field', function() {
     const pdSchema: PresentationDefinition = getFile('./test/dif_pe_examples/pd/pd-simple-schema-age-predicate.json').presentation_definition;
-    const vpSimple: Presentation = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.